projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -511,21 +565,17 @@ export class DatagridComponent<R> implements OnInit {
     }
 
     /**
-     * Resets the pagination to page 1.
+     * Is the given column hideable?
```

Comment:
Probably better to explain what hideable means instead of repeating the text of the method